tokio/src/runtime/mod.rs

```diff
@@ -312,6 +312,19 @@ cfg_rt! {
         blocking_pool: BlockingPool,
     }
 
+    /// The flavor of a `Runtime`.
+    ///
+    /// This is the return type for [`Handle::runtime_flavor`](crate::runtime::Handle::runtime_flavor()).
+    #[derive(Debug, PartialEq, Eq)]
+    #[non_exhaustive]
+    pub enum RuntimeFlavor {
+        /// The flavor that executes all tasks on the current thread.
+        CurrentThread,
+        /// The flavor that executes tasks across multiple threads.
+        #[cfg(all(feature = "rt-multi-thread", not(tokio_wasi)))]
+        MultiThread,
+    }
```

Comment:
Actually, I think it should exist in this enum even if the feature is disabled. Otherwise downstream crates will need their own cfg to compile out the branch.

Reply:
Moved the cfg to the match statement in ``runtime_flavor()``.